backlogger.py

```diff
@@ -44,13 +45,32 @@ def get_json(conf):
     return json.loads(answer.content)
 
 
+def json_rest(method, poo, rest):
+    text = json.dumps(rest)
+    headers = {
+      'User-Agent': 'backlogger ({})'.format(data['url']),
+      'Content-Type': 'application/json',
+      'X-Redmine-API-Key': os.environ['REDMINE_API_KEY'],
+    }
+    r = requests.request(method, '{}/{}.json'.format(data['web'], poo['id']), data=text, headers=headers)
+    r.raise_for_status()
+    return r.json() if r.text else None
+
+
+def issue_reminder(poo):
+    if '--reminder-comment-on-issues' in sys.argv:
+        priority = poo['priority']['name']
+        msg = 'This ticket was set to "{}" priority but was not updated [within the SLO period]({})'.format(priority, data['url'])
```

Comment:
I suggest to include the first reminder from https://progress.opensuse.org/projects/openqatests/wiki#SLOs-service-level-objectives , otherwise people won't know what's expected from them

Reply:
I used the reminder but dropped the period because I couldn't find a way to get that from the query. Do you feel the query as well the the explicit reference to that wiki page is not clear enough?

Reply:
Case in point it looks like this in practice:

This ticket was set to "Immediate" priority but was not updated [within the SLO period](https://progress.opensuse.org/projects/openqatests/wiki#SLOs-service-level-objectives)

Reply:
Oh, I guess you meant to refer to the literal "first reminder". I'll add it